pkg/R/RDD.R

```diff
@@ -1100,9 +1137,7 @@ setMethod("saveAsObjectFile",
             if (!rdd@env$serialized) {
               rdd <- reserialize(rdd)
             }
-            .jcall(getJRDD(rdd), "V", "saveAsObjectFile", path)
-            # NOTE: rJava doesn't check for exceptions if the return type is void
```

Comment:
Just wanna check with you that removing this line in the new patch does not get this regression.